week-4-practise/level-2/exercise-1.js

```diff
@@ -11,12 +11,16 @@ We made a start on this for you here:
 var bottle = {
   volume: 0,
   fill: function() {
+    return (this.volume = 100);

```

Comment:
Correct, but you don't need to return a value for this one or `drink`.

Reply:
sorry i cant understand this !

Reply:
Functions don't always have to return something. You can update the volume without returning a value.

Reply:
ok got it